src/parameterizations/lateral/MOM_tidal_forcing.F90

```diff
@@ -431,7 +610,7 @@ subroutine calc_tidal_forcing(Time, eta, eta_tidal, G, CS, deta_tidal_deta, m_to
     return
   endif
 
-  now = time_type_to_real(Time)
+  now = time_type_to_real(Time) - cs%time_ref
```

Comment:
CS%time_ref should be a time_type, and the difference should be taken between time_types before conversion to a real number of seconds.